### PR TITLE
Add/fix/tweak translated strings

### PR DESCRIPTION
**Pull request by UdjinM6**
Pls see individual commits

### CODE REVIEW

File: src/coinjoin/coinjoin.cpp [hunk 1]
```diff
@@ -499,7 +499,7 @@ std::string CCoinJoin::GetMessageByID(PoolMessage nMessageID)
     case ERR_QUEUE_FULL:
         return _("Masternode queue is full.");
     case ERR_RECENT:
-        return _("Last CoinJoin was too recent.");
+        return _("Last queue was too recent.");
```
Comment (xdustinface) [inline src/coinjoin/coinjoin.cpp:502]:
```suggestion
        return _("Last queue was created too recent.");
```

File: src/qt/forms/optionsdialog.ui [hunk 2]
```diff
@@ -278,10 +278,10 @@
           <item>
            <widget class="QCheckBox" name="coinJoinEnabled">
             <property name="toolTip">
-             <string>Show mixing interface on Overview screen and reveal CoinJoin screen which allows to spend fully mixed coins only.&lt;br/&gt;A new tab with more settings will also appear in this dialog, please make sure to check them before mixing your coins.</string>
+             <string>Show mixing interface on Overview screen and reveal additional screen which allows to spend fully mixed coins only.&lt;br/&gt;A new tab with more settings will also appear in this dialog, please make sure to check them before mixing your coins.</string>
```
Comment (xdustinface) [inline src/qt/forms/optionsdialog.ui:281]:
```suggestion
             <string>Show mixing interface on Overview screen and reveal an additional screen which allows to spend fully mixed coins only.&lt;br/&gt;A new tab with more settings will also appear in this dialog, please make sure to check them before mixing your coins.</string>
```

File: src/qt/forms/optionsdialog.ui [hunk 3]
```diff
@@ -307,40 +307,40 @@
...
-          <string>Show popups for CoinJoin transactions</string>
+          <string>Show popups for mixing transactions</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="lowKeysWarning">
          <property name="toolTip">
-          <string>Show warning dialog when CoinJoin detects that wallet has very low number of keys left.</string>
+          <string>Show warning dialog when wallet has very low number of keys left.</string>
```
Comment (xdustinface) [inline src/qt/forms/optionsdialog.ui:330]:
```suggestion
          <string>Show warning dialog when the wallet has very low number of keys left.</string>
```

File: src/qt/forms/optionsdialog.ui [hunk 4]
```diff
@@ -307,40 +307,40 @@
...
          </property>
          <property name="text">
-          <string>Warn if CoinJoin is running out of keys</string>
+          <string>Warn if wallet is running out of keys</string>
```
Comment (xdustinface) [inline src/qt/forms/optionsdialog.ui:333]:
```suggestion
          <string>Warn if the wallet is running out of keys</string>
```

File: src/qt/forms/optionsdialog.ui [hunk 5]
```diff
@@ -352,7 +352,7 @@
             <string>This setting determines the amount of individual masternodes that an input will be mixed through.&lt;br/&gt;More rounds of mixing gives a higher degree of privacy, but also costs more in fees.</string>
            </property>
            <property name="text">
-            <string>CoinJoin rounds to use</string>
+            <string>Rounds to use</string>
```
Comment (xdustinface) [inline src/qt/forms/optionsdialog.ui:355]:
```suggestion
            <string>Mixing rounds</string>
```

File: src/wallet/wallet.cpp [hunk 6]
```diff
@@ -3758,10 +3758,10 @@ bool CWallet::CreateTransaction(const std::vector<CRecipient>& vecSend, CTransac
                     std::set<CInputCoin> setCoinsTmp;
                     if (!SelectCoins(vAvailableCoins, nValueToSelect, setCoinsTmp, nValueIn, coin_control, coin_selection_params, bnb_used)) {
                         if (coin_control.nCoinType == CoinType::ONLY_NONDENOMINATED) {
-                            strFailReason = _("Unable to locate enough CoinJoin non-denominated funds for this transaction.");
+                            strFailReason = _("Unable to locate enough non-denominated funds for this transaction.");
                         } else if (coin_control.nCoinType == CoinType::ONLY_FULLY_MIXED) {
-                            strFailReason = _("Unable to locate enough CoinJoin denominated funds for this transaction.");
-                            strFailReason += " " + _("CoinJoin uses exact denominated amounts to send funds, you might simply need to mix some more coins.");
+                            strFailReason = _("Unable to locate enough denominated funds for this transaction.");
```
Comment (xdustinface) [inline src/wallet/wallet.cpp:3763]:
```suggestion
                            strFailReason = _("Unable to locate enough mixed funds for this transaction.");
```

File: src/wallet/wallet.cpp [hunk 7]
```diff
@@ -3758,10 +3758,10 @@ bool CWallet::CreateTransaction(const std::vector<CRecipient>& vecSend, CTransac
...
+                            strFailReason += " " + strprintf(_("%s uses exact denominated amounts to send funds, you might simply need to mix some more coins."), "CoinJoin");
```
Comment (xdustinface) [inline src/wallet/wallet.cpp:3764]:
```suggestion
                            strFailReason += " " + strprintf(_("%s uses exact mixed amounts to send funds, you might simply need to mix some more coins."), "CoinJoin");
```

**Review by xdustinface — CHANGES_REQUESTED** [review]
See below for some potential changes

File: src/wallet/wallet.cpp (hunk 7 above)
Comment (UdjinM6) [reply src/wallet/wallet.cpp:3764]: hmm, no, the idea here was to point out that we use denominations, can't just pick some random amounts and send it

Comment (UdjinM6) [reply src/wallet/wallet.cpp:3764]:
```suggestion
                            strFailReason += " " + strprintf(_("%s uses exact denominations to send funds, you might simply need to mix some more coins."), "CoinJoin");
```
maybe?

File: src/wallet/test/wallet_tests.cpp [hunk 8]
```diff
@@ -385,8 +385,8 @@ class CreateTransactionTestSetup : public TestChain100Setup
     const std::string strTooSmallToPayFee = "The transaction amount is too small to pay the fee";
     const std::string strTooSmallAfterFee = "The transaction amount is too small to send after the fee has been deducted";
     const std::string strTooSmall = "Transaction amount too small";
-    const std::string strUnableToLocateCoinJoin1 = "Unable to locate enough CoinJoin non-denominated funds for this transaction.";
-    const std::string strUnableToLocateCoinJoin2 = "Unable to locate enough CoinJoin denominated funds for this transaction. CoinJoin uses exact denominated amounts to send funds, you might simply need to mix some more coins.";
+    const std::string strUnableToLocateCoinJoin1 = "Unable to locate enough non-denominated funds for this transaction.";
+    const std::string strUnableToLocateCoinJoin2 = "Unable to locate enough denominated funds for this transaction. CoinJoin uses exact denominated amounts to send funds, you might simply need to mix some more coins.";
```
Comment (xdustinface) [inline src/wallet/test/wallet_tests.cpp:389]:
```suggestion
    const std::string strUnableToLocateCoinJoin2 = "Unable to locate enough mixed funds for this transaction. CoinJoin uses exact denominated amounts to send funds, you might simply need to mix some more coins.";
```

Ops, here too then

Comment (UdjinM6) [reply src/wallet/test/wallet_tests.cpp:389]: nah, this one makes sense imo - you do need _mixed funds_ in _denominated amounts_ ;)

File: src/wallet/wallet.cpp (hunk 7 above)
Comment (xdustinface) [reply src/wallet/wallet.cpp:3764]: Ah right, well so im fine either way. Initial or your suggestion :)

**Review by xdustinface — DISMISSED** [review]
utACK

File: src/coinjoin/coinjoin.cpp [hunk 9]
```diff
@@ -499,7 +499,7 @@ std::string CCoinJoin::GetMessageByID(PoolMessage nMessageID)
     case ERR_QUEUE_FULL:
         return _("Masternode queue is full.");
     case ERR_RECENT:
-        return _("Last queue was too recent.");
+        return _("Last queue was created too recent.");
```
Comment (thephez) [inline src/coinjoin/coinjoin.cpp:502]:
```suggestion
        return _("Last queue was created too recently.");
```

**Review by thephez — COMMENTED** [review]
One minor suggestion

**Review by PastaPastaPasta — APPROVED** [review]
utACK

**Review by thephez — APPROVED** [review]
Very-slightly tested ACK

**Review by xdustinface — APPROVED** [review]
utACK

**Pull request merged**